src/CMakeLists.txt

```diff
@@ -33,12 +33,15 @@ add_library(nats SHARED ${SOURCES} ${PS_SOURCES} ${S_SOURCES})
 target_link_libraries(nats ${NATS_OPENSSL_LIBS} ${NATS_EXTRA_LIB} ${NATS_PROTOBUF_LIB})
 add_library(nats_static STATIC ${SOURCES} ${PS_SOURCES} ${S_SOURCES})
 target_link_libraries(nats_static ${NATS_OPENSSL_LIBS} ${NATS_PROTOBUF_LIB})
+set_target_properties(nats nats_static PROPERTIES
+  VERSION ${NATS_VERSION_MAJOR}.${NATS_VERSION_MINOR}.${NATS_VERSION_PATCH}
+  SOVERSION ${NATS_VERSION_MAJOR}.${NATS_VERSION_MINOR})
 
 # --------------------------------------
 # Install the libraries and header files
 # --------------------------------------
-install(TARGETS nats DESTINATION lib)
-install(TARGETS nats_static DESTINATION lib)
-install(FILES deprnats.h DESTINATION include RENAME nats.h)
-install(FILES nats.h status.h version.h DESTINATION include/nats)
-install(FILES adapters/libevent.h adapters/libuv.h DESTINATION include/nats/adapters)
+install(TARGETS nats DESTINATION ${NATS_LIBDIR})
```

Comment:
I gather than removing `LIBRARY` works (install the shared lib in the appropriate directory) on your platform of choice, right?

Reply:
@kozlovic Yes, all is working as expected.